### ISSUE TITLE
Detect if mailmen are the ones placing the box

### ISSUE THREAD

**Issue by jchoyt** [jchoyt/WoolfPostalService#4]
Detect mailmen and don't report out if they are the ones placing the delivery.

Should we announce the delivery instead?

**Comment by Woolf622** [comment]
yes. how about if mailmen place a package. it would send a message to that player when they get online. 
"WPS has delivered a package for <nickname>". 
if i was to make all the nicknames the exact ign of people would that work?

**Comment by jchoyt** [comment]
That would certainly be possible.  Let me think if there's an easier way (for you).

**Comment by jchoyt** [comment]
See https://hub.spigotmc.org/javadocs/bukkit/org/bukkit/permissions/Permissible.html and hasPermission(String s)
